aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -507,236 +537,15 @@ where
     }
 
     fn skip_output() -> Self {
-        Self(vec![], vec![], vec![], OnceCell::new())
+        Self(vec![], vec![], vec![], OnceCell::new(), 0)
```

Comment:
A general question / suggestion. Some IDEs show what are these fields, but some do not. I sometimes use inline comments to make sure it is easy to see what the constants refer to, e.g. 
```rust
Self(/*writes=*/vec![], ..., /*gas=*/0)
```
Not sure what is the Rust standard for this.

Reply:
I sometimes add trailing comments too, let me see what I can do.
And Output I'll refactor more in later PRs :-)